ragna/source_storages/_vector_database.py

```diff
@@ -61,78 +17,4 @@ def requirements(cls) -> list[Requirement]:
             # requirements. We use this as base for all of our Vector DBs.
             PackageRequirement("chromadb>=0.4.13"),
             PackageRequirement("tiktoken"),
```

Comment:
They shouldn't be needed anymore given that this now happens on the `EmbeddingModel`.

Reply:
Yes this entire class can be removed now actually, if you don't mind.  It doesn't really have to wait until the chunking PR.

Reply:
Note that there is https://github.com/Quansight/ragna/pull/369#discussion_r1537222442. I don't mind making that a standalone function in a `_utils` module though.